server/routes/index.js

```diff
@@ -2,10 +2,10 @@ var express = require('express');
 var router = express.Router();
 const authRoutes = require('./auth');
 const userRoutes = require('./users');
-const projectRoutes = require('./Project');
+const projectRoutes = require('./project');
```

Comment:
need to rename the server/routes/Project to /project.js

Reply:
done